### DISCUSSION TITLE
Tips requested for developing filecoin

### DISCUSSION THREAD

**Discussion by CullerWhale** [filecoin-project/community#374] [Q&A]
Hello, 
I am a complete noob that would like to work to help develop filecoin. My first step is to earn a certification from a top university full stack bootcamp. I want to enquire your thoughts on what else I could be doing to get hired by a blockchain development company/organization, or to help contribute to projects like filecoin. 

Thank you

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx
